perf/common.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+exports.createMsg = function(msgType, sz) {
+    var buf;
+    switch (msgType) {
+        case '--buffer':
+            buf = new Buffer(sz);
+            buf.fill('o');
+            break;
+        case '--string':
+            buf = new Array(sz + 1).join('o');
+            break;
+        default:
+            console.err('Buffer type not supported please use --buffer or --string.');
```

Comment:
can we say `Unspecified msg type not supported, please use --buffer or --string`